contrib/dracut/90zfs/Makefile.am

```diff
@@ -9,6 +9,9 @@ pkgdracut_SCRIPTS = \
 	zfs-needshutdown.sh \
 	zfs-lib.sh
 
+pkgdracut_DATA = \
+	zfs-dracut-bootfs.service.in
```

Comment:
Putting this in `DATA` makes sense, but I think you need to remove the `.in` suffix since we want to install the generated version.  Similar to `SCRIPTS` above.

Reply:
Oops... copy'n'paste mistake..